test/VeresOneDriver.spec.js

```diff
@@ -173,6 +176,23 @@ describe('methods/veres-one', () => {
       expect(didDocument.id).to.match(/^did:v1:nym:z.*/);
     });
 
+    it('should generate a DID document from seed', async () => {
+      const seedBytes = (new TextEncoder()).encode(TEST_SEED).slice(0, 32);
```

Comment:
This is a really interesting way of generating a 32-byte seed. @davidlehn Do we have a recommended way of generating a 32-byte seed from a string? @JSAssassin I don't think this is wrong, but it just got me wondering how we should be turning strings into 32-byte seeds. Also of note: this code comes from the `did-method-key-js` test suite here: https://github.com/digitalbazaar/did-method-key/blob/602b82f8f3c71d687035e5642f3e32fa2651e2d0/test/driver.spec.js#L201-L213